package.json

```diff
@@ -87,29 +83,22 @@
     "identity-obj-proxy": "^3.0.0",
     "jest": "^25.1.0",
     "jest-mock-axios": "^3.1.2",
-    "node-sass": "^5.0.0",
     "npm-run-all": "^4.1.5",
     "prettier": "^2.2.1",
     "redux-mock-store": "^1.5.4",
-    "webpack": "^5.30.0",
-    "webpack-bundle-analyzer": "^4.4.0",
-    "webpack-cli": "^4.6.0",
-    "xhr-mock": "^2.5.1"
+    "webpack-bundle-analyzer": "^4.4.0"
   },
   "scripts": {
     "analyze": "NODE_ENV=production webpack --config config/prod.webpack.config.js --env analyze=true",
     "build": "webpack --config config/dev.webpack.config.js",
     "extract:messages": "npx @formatjs/cli extract 'src/**/*.{js,jsx}' --out-file ./translations/messages.json",
     "lint": "yarn eslint ./src",
-    "prod": "NODE_ENV=production webpack serve --config config/dev.webpack.config.js",
-    "server:ctr": "node src/server/generateServerKey.js",
+    "prod": "NODE_ENV=production webpack serve --config config/prod.webpack.config.js",
     "start": "NODE_ENV=development webpack serve --config config/dev.webpack.config.js",
     "start:beta": "NODE_ENV=development BETA=true webpack serve --config config/dev.webpack.config.js",
-    "start:proxy": "NODE_ENV=development API_ENDOINT=https://ci.cloud.redhat.com webpack serve --config config/dev.webpack.config.js",
     "test": "jest --passWithNoTests --runInBand",
     "build:prod": "NODE_ENV=production webpack --config config/prod.webpack.config.js",
```

Comment:
I believe this one is not used as well. The custom release only calls the `build` script

Reply:
updated